docs/product/integrations/index.mdx

```diff
@@ -20,6 +20,7 @@ For more details, see the [full Integration Platform documentation](/product/int
 | [Microsoft Teams](/product/integrations/notification-incidents/msteams/) | X            | X             |                |
 | [Opsgenie](/product/integrations/notification-incidents/opsgenie/)       | X            | X             |                |
 | [PagerDuty](/product/integrations/notification-incidents/pagerduty/)     | X            | X             |                |
+| [PagerTree](/product/integrations/notification-incidents/pagertree/)     | X            | X             |                |
```

Comment:
This link is broken because the url of the page is actually".../product/integrations/notification-incidents/pagertreeintegration/" which doesn't match with the above :)